openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -684,7 +686,12 @@ def _get_library_component_tags_count(library_key) -> dict:
     return get_object_tag_counts(library_key_pattern, count_implicit=True)
 
 
-def get_library_components(library_key, text_search=None, block_types=None) -> QuerySet[Component]:
+def get_library_components(
+    library_key,
+    text_search=None,
+    block_types=None,
+    in_collections=False,
```

Comment:
Personally I don't think this `in_collections` parameter should be part of this public API interface. It's a very specific thing that I can't really imagine other uses for. See below.

Reply:
Updated: https://github.com/openedx/edx-platform/pull/35734/commits/5e771cebf28e066e462fa39e6dac79d713de79f4